custom_components/tuya_local/devices/eeese_emil_dehumidifier.yaml

```diff
@@ -0,0 +1,187 @@
+name: Eeese Emil Dehumidifier
+products:
+  - id: DYD-S12A431 除湿机
+    name: Eeese Emil Dehumidifier
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  name: Panel
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 1
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 25
+        max: 80
+      mapping:
+        - step: 5
+    - id: 4
+      name: mode
+      type: string
+      mapping:
+        - dps_val: manual
+          value: normal
+        - dps_val: laundry
+          value: boost
+          icon: "mdi:tshirt-crew-outline"
+    - id: 16
+      type: integer
+      name: current_humidity
+    - id: 19
+      type: bitfield
+      name: error
+      mapping:
+        - dps_val: 1
+          icon: "mdi:cup-water"
+          icon_priority: 2
+      hidden: true
+secondary_entities:
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 14
+        type: boolean
+        name: lock
+  - entity: fan
+    translation_key: fan
+    name: Fan
+    category: config
+    dps:
+      - id: 1
+        type: boolean
+        name: switch
+      - id: 4
+        type: string
+        name: dehumidifier_mode
+        hidden: true
+      - id: 5
+        type: string
+        name: speed
+        mapping:
+          - dps_val: low
+            value: 50
+            constraint: dehumidifier_mode
+            conditions:
+              - dps_val: laundry
+                invalid: true
+          - dps_val: high
+            value: 100
+  - entity: sensor
+    class: duration
+    category: diagnostic
+    name: Remaining duration
+    dps:
+      - id: 18
+        type: integer
+        name: sensor
+        unit: "s"
+        class: measurement
```

Comment:
It seems that this value is never updated. I never received anything else than 0, whatever test I tried. 
Should I remove it ? (or leave it if a firmware upgrade fixes it on the device ? )

Reply:
My expectation would be that it only changes after dp 17 is set.

Reply:
It was as well for me. But it does not happen. Whatever I do set on dp17 (or on the physical device) this one does not move from `0`. So I'll comment it, in case later they fix it on the device itself.

Reply:
Rather than comment it, move it to an attribute of some other entity, so it is still considered in device matching, and still available if someone else's device is working differently.